client/src/test/java/model/DummyClickableField.java

```diff
@@ -0,0 +1,48 @@
+package model;
+
+
+import gui.fields.ClickableField;
+
+class DummyClickableField implements ClickableField {
+
+  private final Integer row;
+  private final Integer col;
+
+  DummyClickableField(Integer row, Integer col) {
+    this.row = row;
+    this.col = col;
+  }
+
+  DummyClickableField(int position) {
+    int boardSize = 10;
+    this.col = position % boardSize;
+    this.row = position / boardSize;
+  }
+
+  @Override
+  public void makeUnclickable() {
+  }
+
+  @Override
+  public void makeClickable() {
+  }
```

Comment:
why this methods are empty?

Reply:
This is just dummy implementation for testing, my own mock :), we dont need body for this method, becouse we dont test ClickableField, but we use this implementation for testing other classes.